MapboxMobileEvents/Categories/UIKit+MMEMobileEvents.m

```diff
@@ -82,3 +82,46 @@ - (NSInteger)mme_contentSizeScale {
 }
 
 @end
+
+@implementation NSExtensionContext (MMEMobileEvents)
+
++ (NSInteger)mme_contentSizeScale {
+
+    NSInteger result = -9999;
+
+    if (@available(iOS 10, *)) {
+        NSString *sc = UIScreen.mainScreen.traitCollection.preferredContentSizeCategory;
```

Comment:
Given UIApplication shared isn't available, grab preferredContentSizeCategory directly from UIScreen. Unfortunately this only became available in iOS10. Based on prior result of -9999 value, assumed this same default would be used in the event user is running < iOS10